src/main/java/uk/ac/ebi/ena/webin/cli/WebinCliExecutor.java

```diff
@@ -128,6 +130,32 @@ private void setIgnoreErrors(M manifest) {
         }
     }
 
+    private void checkGenomeSubmissionRatelimit() {
+        if (getContext().equals(WebinCliContext.genome)) {
+            M manifest = getManifestReader().getManifest();
+            boolean ratelimit = false;
+
+            try {
+                RatelimitService ratelimitService = new RatelimitService.Builder()
+                        .setCredentials(getParameters().getWebinServiceUserName(),
+                                getParameters().getPassword())
+                        .setTest(getParameters().isTest())
+                        .build();
+
+                String subAccId = getParameters().getWebinServiceUserName();
+                String studyId = manifest.getStudy().getStudyId();
+                String sampleId = manifest.getSample().getSraSampleIdId();
+
+                ratelimit = ratelimitService.ratelimit(getContext().name(), subAccId, studyId, sampleId);
+            } catch (RuntimeException ex) {
+                log.warn(WebinCliMessage.RATE_LIMIT_SERVICE_SYSTEM_ERROR.text());

```

Comment:
- I would throw an exception instead (and trust the new ratelimit endpoint to work):
```
throw WebinCliException.systemError(WebinCliMessage.RATE_LIMIT_SERVICE_SYSTEM_ERROR.text());
```